pkgs/servers/web-apps/livebook/default.nix

```diff
@@ -0,0 +1,45 @@
+# Created with inspiration from https://github.com/hauleth/nix-elixir/blob/master/pkgs/livebook.nix
```

Comment:
I think you should remove that comment and add it in the PR only.

Reply:
Hey thanks for coming by !
If you dont mind i would like to keep the reference.
First as a thank you, but mostly because its hard to find repos for nix and elixir and giving a reference might provide help to people looking for solutions to obscure compilation problems with nix and elixir.

Reply:
I understand, it's just that I haven't found this yet in `nixpkgs`, so I'm wondering if this is a good practice.

Reply:
I also think it would be nice to keep, for the same reason @happysalada mentioned, but if you think it's poor form I'll remove it.

Reply:
Also it might help future maintainers in case something breaks here, upstream might have fixes.

There is another occurence in nixpkgs, from the same repo actually ( i wrote it). I put it because we struggled so much around an issue and upstream provided great hints at the solution.

Personally i think comments are there to help developpers, so anything going in that direction is faire game.

I would suggest to keep it, but if there is policy somewhere that emerges against it, then we can definitely remove it.

Reply:
Fair enough :)

Reply:
Thank you for your understanding on this !

Reply:
The linked file is using nix code that is not in line with our coding standard. Also this file is seemingly a very basic mix build making the link not that useful. So the link is most likely only going to rot over time and becoming less relevant. Actually right now it points to version 0.8.0 while we are adding 0.9.2.

```suggestion
```